.github/workflows/docker.yaml

```diff
@@ -0,0 +1,55 @@
+name: Build And Publish localdev-server Docker Image
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.commits'
+      - '*.markdown'
+      - '**/*.markdown'
+
+env:
+  LC_ALL: en_US.UTF-8
+  IMAGE_NAME: localdev-server
+  DOCKER_USER: ${{ secrets.DOCKERHUB_USERNAME }}
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  BuildDockerImage:
+    if: ${{ (github.repository == 'rotty3000/lxc-localdev') }}
+    name: Build Docker Image
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        persist-credentials: false
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
```

Comment:
Just out of curiosity, what does this do?

Reply:
`buildx` will fail on non-native architectures if it doesn't find architecture emulation.